api.js

```diff
@@ -1044,6 +1131,30 @@
         params.token = this.token;
       }
 
+      if (cachedRoutes[lcMethod]) {
+        var i;
+        for(i in cachedRoutes[lcMethod]) {
+          if (!cachedRoutes[lcMethod].hasOwnProperty(i)) continue;
+          var regexp = new RegExp(i);
```

Comment:
Only storing the regexp is required for the browser to cache a compiled regex. Using `.compile()` will trigger all regexes to compile on 'boot' instead of their first call.

http://www.levihackwith.com/javascript-performance-tip-precompile-your-regular-expressions/